src/controllers/v1/did/DidController.ts

```diff
@@ -28,18 +29,27 @@ export class DidController extends Controller {
   @Response<BadRequest['message']>(400)
   @Response<HttpResponse>(500)
   public async getCredentials(
```

Comment:
I'm guessing I copy pasted this when I implemented did listing. Can we rename this methiod please to `listDids` or similar whilst I notice it

Reply:
thank for a flag up, did not even noticed. updated.